src/core/EdFi.DataManagementService.Core.External/Model/IBaseResourceInfo.cs

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.DataManagementService.Core.External.Model;
+
+/// <summary>
+/// Base API resource information for passing along to backends.
+/// </summary>
+public interface IBaseResourceInfo
```

Comment:
I wouldn't say I'm thrilled with all these duplicative interfaces, but it's better that some sort of object mapping-ish scheme and seems C#-ish enough. Can revisit once C# gets duck typing. 😋